src/main/java/org/example/MercedesSKlasse.java

```diff
@@ -0,0 +1,53 @@
+package org.example;
+
+public class MercedesSKlasse extends Mercedes{
+    public MercedesSKlasse(float fuelTankSize, int gears, String fuelType, int tireSize, String chassisNumber) {
+        super(fuelTankSize, gears, fuelType, tireSize, chassisNumber);
+    }
+    /**
+     * Method starts the engine and sets the initial fuel consumption based on tyre size.
+     */
+    @Override
+    public void start() {
+        super.start();
+        System.out.println("The Mercedes SKlasse started.");
```

Comment:
same comment as the other class, make a generic fuel consumpion method which takes a parameter, the car's normal consumption

Reply:
and here